web/src/main/kotlin/no/nav/su/se/bakover/web/routes/behandling/BehandlingRoutes.kt

```diff
@@ -194,7 +193,7 @@ internal fun Route.behandlingRoutes(
                                 {
                                     when (it) {
                                         is AttestantOgSaksbehandlerErLik -> call.svar(Forbidden.message(it.msg))
-                                        is Utbetaling -> call.svar(InternalServerError.message(it.msg))
+                                        else -> call.svar(InternalServerError.message(it.msg))
```

Comment:
Foreslår og ha en exhaustive when, og heller bruke commaseparerte lister is Utbetaling, isAttesta.... ->

Da får man compilefeil når man legger til nye og må da ta et valg